teloszkevm.tokenlist.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "TelosZKEVM Token List",
-  "logoURI": "https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/telos.png",
+  "logoURI": "https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/telos-zkevm-logo-256.png",
```

Comment:
This line, currently is https://github.com/telosnetwork/token-list/blob/a996965f0413f2d1ab2bd78c5c0e558d1be1a999/logos/telos-zkevm-logo-256.png
But once it's merged it will become https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/telos-zkevm-logo-256.png which currently does not exist